webpages/settings/components/category-selector.html

```diff
@@ -40,7 +40,6 @@
   }
 
   .category.sel {
-    color: var(--orange);
     font-weight: var(--brand-orange-min-font-weight);
```

Comment:
```suggestion
    font-weight: 600;
```
This CSS variable specifies the font weight of orange text. It shouldn't be used here.